### PR TITLE
Feat/improve performance analysis

### PR DESCRIPTION
**Pull request by anish-neervannan**
**Before submitting this PR, please make sure**:
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/", etc)

**Optional**:
- Related Github issue:
- Related Clubhouse Story:

Added more information to performance analysis feature, which now prints out starting balance, current balance, delta, and returns. Added more unit tests to test these new features out. Also updated documentation to link to the explanation of how the returns are calculated.

### CODE REVIEW

File: hummingbot/client/performance_analysis.py [hunk 1]
```diff
@@ -51,11 +51,30 @@ def add_balances(self, asset_name: str, amount: float, is_base: bool, is_startin
...
 
-    def compute_profitability(self, price: float) -> float:
-        """ Compute the profitability of the trading bot based on the starting and current prices """
+    def compute_starting(self, price: float) -> (str, float):
+        """ Computes the starting amount of token between both exchanges. """
         starting_amount = (self._starting_base.amount * price) + self._starting_quote.amount
+        starting_token = self._starting_quote.token
+        return starting_token, starting_amount
+
+    def compute_current(self, price: float) -> (str, float):
```
Comment (chowryan) [inline hummingbot/client/performance_analysis.py]: -> Tuple[str, float]:

File: hummingbot/client/performance_analysis.py [hunk 2]
```diff
@@ -51,11 +51,30 @@ def add_balances(self, asset_name: str, amount: float, is_base: bool, is_startin
...
+        """ Computes the current amount of token between both exchanges. """
+        current_amount = (self._current_base.amount * price) + self._current_quote.amount
+        current_token = self._current_quote.token
+        return current_token, current_amount
+
+    def compute_delta(self, price: float) -> (str, float):
```
Comment (chowryan) [inline hummingbot/client/performance_analysis.py]: -> Tuple[str, float]:

File: hummingbot/client/performance_analysis.py [hunk 3]
```diff
@@ -51,11 +51,30 @@ def add_balances(self, asset_name: str, amount: float, is_base: bool, is_startin
                 temp_amount = erc.convert_token_value(amount, asset_name, currency_amount.token)
                 currency_amount.amount += temp_amount
 
-    def compute_profitability(self, price: float) -> float:
-        """ Compute the profitability of the trading bot based on the starting and current prices """
+    def compute_starting(self, price: float) -> (str, float):
```
Comment (chowryan) [inline hummingbot/client/performance_analysis.py]: -> Tuple[str, float]:

**Comment by chowryan** [comment]
nice tests!

**Comment by anish-neervannan** [comment]
Made minor changes suggested by @chowryan. I think we are good to merge this @mifeng

**Pull request merged**